<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-UCNIV3S" failures="1" tests="8" name="Test.HomeTest" time="42.243" errors="0" timestamp="2022-01-31T13:43:52 IST" skipped="0">
  <testcase classname="Test.HomeTest" name="base" time="0.036"/>
  <system-out/>
  <testcase classname="Test.HomeTest" name="rememberedPassword" time="4.115"/>
  <system-out/>
  <testcase classname="Test.HomeTest" name="verifyGetYourAccountPage" time="2.774"/>
  <system-out/>
  <testcase classname="Test.HomeTest" name="verifyhomePagetitle" time="19.372"/>
  <system-out/>
  <testcase classname="Test.HomeTest" name="verifydate" time="4.041">
    <failure message="date didnt match expected [ Saturday 29 January-2022] but found [ Monday 31 January-2022]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: date didnt match expected [ Saturday 29 January-2022] but found [ Monday 31 January-2022]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at Test.HomeTest.verifydate(HomeTest.java:63)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifydate -->
  <system-out/>
  <testcase classname="Test.HomeTest" name="verifyAttendanceMark" time="6.983"/>
  <system-out/>
  <testcase classname="Test.HomeTest" name="verifyClockIn" time="4.527"/>
  <system-out/>
  <testcase classname="Test.HomeTest" name="verifySearchMenu" time="0.395"/>
  <system-out/>
</testsuite> <!-- Test.HomeTest -->
